contributing-guides/alias-pages.md

```diff
@@ -115,7 +115,15 @@ Not translated yet.
 
 ### ml
 
-Not translated yet.
+```Markdown
+# ഉദാഹരണം
+
+> ഈ കമാൻഡ് `ഉദാഹരണം` എന്നത്തിന്റെ അപരനാമമാണ്.
+
+- യഥാർത്ഥ കമാൻഡിനായി ഡോക്യുമെന്റേഷൻ കാണുക:
+
+`tldr ഉദാഹരണം`
+```
```

Comment:
@marchersimon can you apply this suggestion?
```suggestion
```markdown
# example

> ഈ കമാൻഡ് `example` എന്നത്തിന്റെ അപരനാമമാണ്.

- യഥാർത്ഥ കമാൻഡിനായി ഡോക്യുമെന്റേഷൻ കാണുക:

`tldr example`
```
```